src/quantum/azext_quantum/_help.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
```

Comment:
I suggest write help entry for each command. You only have help entry for command group.